Lesson26/Gokhan/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Image Gallery</title>
+  <link rel="stylesheet" type="text/css" href="./styles.css">
+</head>
+
+<body>
+
+  <h3>Image Gallery</h3>
+
+  <br>
+
+  <label for="image-count">Image Count</label>
```

Comment:
Right not the label looks like a button element which negatively impacts user experience and accessibility. Update the styles so it's easy to understand that 'Image count' is a label, not a button.